src/cmd/manage_nsfs.js

```diff
@@ -49,6 +49,7 @@ function write_stdout_response(response_code, detail, event_arg) {
 const buckets_dir_name = '/buckets';
 const accounts_dir_name = '/accounts';
 const access_keys_dir_name = '/access_keys';
+const acounts_dir_relative_path = '../accounts/';
```

Comment:
What do you think about reusing the string from variable `accounts_dir_name`?

Reply:
`../accounts/` used in multiple places

Reply:
I think we need to move the definitions of the dir names to `manage_nsfs_constants` and the relative as well and use it in multiple places as needed:
https://github.com/noobaa/noobaa-core/blob/e5a28ef5cea44f0f2c9dccd5c2c3a26a762fac1e/src/cmd/manage_nsfs.js#L49-L51

We can do it in a different PR.

Reply:
I totally agree with @shirady - keep the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
I also don't think these const dir names should have `/` in the beginning, this is just confusing and appears like a root FS path. Instead, using path.join() will add dir separators as needed. But we can defer to next PR to avoid delaying this fix.

Reply:
@guymguym @shirady sure will add this changes in my next PR